sync_diff_inspector/diff.go

```diff
@@ -157,6 +167,41 @@ func (df *Diff) AdjustTableConfig(cfg *Config) error {
 				continue
 			}
 
+			sourceTables := make([]TableInstance, 0, 1)
+			// find matched datbase name and table name in table router to fill source tables
+			for instanceID, allSchemas := range allTablesMap {
+				if instanceID == df.targetDB.InstanceID {
+					continue
+				}
+
+				for schema, allTables := range allSchemas {
+					for table := range allTables {
+						targetSchema, targetTable, err := df.tableRouter.Route(schema, table)
+						if err != nil {
+							return errors.Errorf("get route result for %s.%s.%s failed, error %v", instanceID, schema, table, err)
+						}
+
+						if targetSchema == schemaTables.Schema && targetTable == tableName {
+							sourceTables = append(sourceTables, TableInstance{
+								InstanceID: instanceID,
+								Schema:     schema,
+								Table:      table,
+							})
+							log.Infof("find matched table %s.%s.%s with %s.%s.%s", instanceID, schema, table, df.targetDB.InstanceID, schemaTables.Schema, tableName)
+						}
+					}
+				}
+			}
+
+			// use same database name and table name
+			if len(sourceTables) == 0 {
```

Comment:
continue directly？

Reply:
if don't find matched database and table in router, will use same database and table name as default source.